release_notes/ocp-4-2-release-notes.adoc

```diff
@@ -0,0 +1,890 @@
+
+Use the link:https://access.redhat.com/labs/ocplimitscalculator/[{product-title}
+Limit Calculator] to estimate cluster limits for your environment.
+
+[id="ocp-4-2-developer-experience"]
+=== Developer experience
+
+[id="ocp-4-2-ODO"]
+==== OpenShift Do
+
+OpenShift Do (ODO) is a CLI tool for developers to create, build, and deploy
+applications on OpenShift. ODO is completely client based and requires no server
+within the {product-title} cluster for deployment. It detects changes to local
+code and deploys it to the cluster automatically, giving instant feedback to
+validate changes in real time. It supports multiple languages and frameworks.
+
+[id="ocp-4-2-nodes"]
+=== Nodes
+
+[id="ocp-4-2-crio-support"]
+==== CRI-O support
+
+CRI-O is a kubernetes-specific container engine that tracks and versions
+identical to Kubernetes, simplifying support permutations. Adoption is trivial
+because all existing Docker and OCI containers are supported and run well with
+CRI-O.  CRI-O is a light weight, kubernetes-native, OCI-compatible container
+runtime that is life-cycled and managed by {product-title}. You do not need to
+worry about which container runtime is in use. With {product-title}, it is
+always the right one and it provide a complete implementation of the Kubernetes
+Container Runtime Interface (CRI). Also, you do not need to separately manage
+the container engine. CRI-O has some tuneables that provide control and security
+for CRI-O; they are easily configured in a CRD, and the settings are propagated
+across the cluster.
+
+[id="ocp-4-2-whitelisting-of-sysctls"]
+==== Whitelisting of sysctls configuration
+
+System administrators can whitelist sysctl on a per-node basis. All safe sysctls
+are enabled by default; all unsafe sysctls are disabled by default.See
+xref:../nodes/containers/nodes-containers-sysctls.adoc#nodes-containers-sysctls[Using
+sysctls in containers] for more information.
+
+[id="ocp-4-2-master-nodes-schedulable"]
+==== Master nodes are now schedulable
+
+In {product-title} 4.2, master nodes are schedulable. See
+xref:../nodes/nodes/nodes-nodes-working.adoc[Working with nodes] for more information.
+
+[id="ocp-4-2-infrastructure"]
+=== Infrastructure
+
+[id="ocp-4-2-rhhi-next"]
+==== RHHI.Next
+
+RHHI.Next is a unified platform for container and VM workloads under common
+management control. Storage, compute, and networking is tightly integrated into
+{product-title} for a seamless customer experience. This includes:
+
+* {product-title}
+* OpenShift Container Storage
+* Container-native Virtualization
+* RHEL CoreOS
+* Industry Standard Server Hardware
+* Network Switches (optional)
+
+Benefits:
+
+* Get the infrastructure for your on-premise application development up and running quickly.
+* Hyperconverged infrastructure makes it easier to order, install, and manage.
+* Run virtualized workloads side-by-side with newer container workloads.
+* Consistent operational management with public cloud using {product-title}.
+
```

Comment:
I guess the RHHI.next section may need to be removed after the recent announcements